test/consistent-function-scoping.js

```diff
@@ -234,12 +250,6 @@ ruleTester.run('consistent-function-scoping', rule, {
 			`,
 			errors: [arrowError]
 		},
-		{
-			code: outdent`
-				const doFoo = () => bar => bar;
-			`,
-			errors: [arrowError]
-		},
```

Comment:
I guess this test case mean to enforce use this style?

```
const movedToOuter = bar => bar;
const doFoo = () => movedToOuter;
```

seems make sense, I'm not doing it right.